dll/win32/msgina/shutdown.c

```diff
@@ -336,6 +348,17 @@ ShutdownOnInit(
             SendMessageW(hwndList, CB_SETITEMDATA, idx, WLX_SAS_ACTION_SHUTDOWN_REBOOT);
     }
 
+    /* Restart in NT Native Mode */
+    if (pContext->ShutdownOptions & WLX_SHUTDOWN_STATE_NT_MODE)
+    {
+        LoadStringW(pgContext->hDllInstance, IDS_SHUTDOWN_NT_MODE, szBuffer, _countof(szBuffer));
+        idx = SendMessageW(hwndList, CB_ADDSTRING, 0, (LPARAM)szBuffer);
+        if (idx != CB_ERR)
+        {
+            SendMessageW(hwndList, CB_SETITEMDATA, idx, WLX_SAS_ACTION_SHUTDOWN_NT_MODE);
+        }
+    }
+
     // if (pContext->ShutdownOptions & 0x08) {}
```

Comment:
This is you. Remove please.

Reply:
N/A due to SAS_ACTION value changed to 0x100 (see comment below)